airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -311,6 +331,80 @@ def execute(self, context: Context):
             subnets=cast(List[str], self.resources_vpc_config.get("subnetIds")),
         )
 
+    def deferrable_create_cluster_next(self, context: Context, event: dict[str, Any] = {}) -> None:
```

Comment:
I'm unsure whether we should use `{}` as the default value. I believe it's generally not a good idea to use mutable object as default

Reply:
hmm that's a good point. I guess I can set it to `None` and do a check for `None` before indexing it i.e.
```
if event and event["status"]
```
Its going to be tedious to do it everywhere, but I don't see a better option

Reply:
Yep, I think this is the best option we have as of now.